### PR TITLE
Last Commit is the one that matters

### PR DESCRIPTION
**Pull request by Elyriand21**
** ADDITIONS **
- Added 'is_drunk' to 'get' command

### CODE REVIEW

File: DeveloperConsole.cs [hunk 1]
```diff
@@ -109,6 +119,11 @@ private void CommandRunner(string command)
                             result = "Error: Player not found";
                             break;
                         }
+                        if (args.Length == 2)
```
Comment (Lillious) [inline DeveloperConsole.cs]: Not Needed

File: DeveloperConsole.cs [hunk 2]
```diff
@@ -51,6 +52,12 @@ private void CommandRunner(string command)
                     }
                     switch (args[1])
                     {
+                        // Help section for 'help'
+                        // Displays the second page of 'help'
+                        case "2":
```
Comment (Lillious) [inline DeveloperConsole.cs]: Not sure what this is for

**Review by Lillious — CHANGES_REQUESTED** [review]

**Review by Elyriand21 — COMMENTED** [review]
Changes committed

**Review by Lillious — APPROVED** [review]
Looks good

**Pull request merged**